extension.js

```diff
@@ -157,6 +157,28 @@ function unsetColor(context, title) {
   }
 }
 
+async function clearAllTabColors(context) {
+  const storage = new Storage(context);
+  const tabs = storage.get("tabs") || {};
+  const editors = vscode.workspace.textDocuments;
+  for (const editor of editors) {
+      const resource = editor.uri;
+      const fileName = resource.fsPath.replace(/\\/g, "\\\\");
+      for (const color in tabs) {
+          const tabIndex = tabs[color].indexOf(fileName);
```

Comment:
use includes rather than indexOf for more readability

Reply:
Function updated and tested to use includes instead.